ambry-api/src/test/java/com.github.ambry/account/InMemAccountService.java

```diff
@@ -149,6 +148,20 @@ public synchronized Account createAndAddRandomAccount() {
     return account;
   }
 
+  /**
+   * Adds {@code replicationPolicy} to {@code container}.
+   * @param container the {@link Container} to add the replication policy to.
+   * @param replicationPolicy the replication policy to add to {@code container}.
+   * @return the new container object with the appropriate replication policy if the operation suceeded. {@code null}
+   * otherwise
+   */
+  public Container addReplicationPolicyToContainer(Container container, String replicationPolicy) {
+    Container newContainer = new ContainerBuilder(container).setReplicationPolicy(replicationPolicy).build();
+    Account account = getAccountById(container.getParentAccountId());
+    Account newAccount = new AccountBuilder(account).addOrUpdateContainer(newContainer).build();
+    return updateAccounts(Collections.singleton(newAccount)) ? newContainer : null;
+  }
```

Comment:
Right - in this implementation, it does. This piece of code is not making the assumption in case the impl of `updateAccounts()` changes in the future.